test/test_automated_install.py

```diff
@@ -120,101 +154,238 @@ def test_configureFirewall_no_firewall(Pihole):
+    check_selinux = Pihole.run('''
+    source /opt/pihole/basic-install.sh
+    checkSelinux
+    ''')
+    expected_stdout = info_box + ' SELinux mode detected: Permissive'
+    assert expected_stdout in check_selinux.stdout
+    assert check_selinux.rc == 0
+
+
+def test_selinux_disabled(Pihole):
+    '''
+    confirms installer continues when SELinux is Disabled
+    '''
+    mock_command('getenforce', {'*': ('Disabled', '0')}, Pihole)
+    check_selinux = Pihole.run('''
+    source /opt/pihole/basic-install.sh
+    checkSelinux
+    ''')
+    expected_stdout = info_box + ' SELinux mode detected: Disabled'
+    assert expected_stdout in check_selinux.stdout
+    assert check_selinux.rc == 0
+
 
 def test_installPiholeWeb_fresh_install_no_errors(Pihole):
-    ''' confirms all web page assets from Core repo are installed on a fresh build '''
+    '''
+    confirms all web page assets from Core repo are installed on a fresh build
+    '''
     installWeb = Pihole.run('''
     source /opt/pihole/basic-install.sh
     installPiholeWeb
     ''')
-    assert info_box + ' Installing blocking page...' in installWeb.stdout
-    assert tick_box + ' Creating directory for blocking page, and copying files' in installWeb.stdout
-    assert cross_box + ' Backing up index.lighttpd.html' in installWeb.stdout
-    assert 'No default index.lighttpd.html file found... not backing up' in installWeb.stdout
-    assert tick_box + ' Installing sudoer file' in installWeb.stdout
+    expected_stdout = info_box + ' Installing blocking page...'
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = tick_box + (' Creating directory for blocking page, '
+                                  'and copying files')
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = cross_box + ' Backing up index.lighttpd.html'
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = ('No default index.lighttpd.html file found... '
+                       'not backing up')
+    assert expected_stdout in installWeb.stdout
+    expected_stdout = tick_box + ' Installing sudoer file'
+    assert expected_stdout in installWeb.stdout
     web_directory = Pihole.run('ls -r /var/www/html/pihole').stdout
     assert 'index.php' in web_directory
     assert 'blockingpage.css' in web_directory
 
+
 def test_update_package_cache_success_no_errors(Pihole):
-    ''' confirms package cache was updated without any errors'''
+    '''
+    confirms package cache was updated without any errors
+    '''
     updateCache = Pihole.run('''
     source /opt/pihole/basic-install.sh
     distro_check
     update_package_cache
     ''')
-    assert tick_box + ' Update local cache of available packages' in updateCache.stdout
+    expected_stdout = tick_box + ' Update local cache of available packages'
+    assert expected_stdout in updateCache.stdout
     assert 'Error: Unable to update package cache.' not in updateCache.stdout
```

Comment:
In the same vein as the firewall test comment, exact `not in` checks are easy to get false positives on.  Maybe pull this back to a looser version like
```
assert 'error' not in  updateCache.stdout.lower()